src/burnchains/burnchain.rs

```diff
@@ -225,6 +225,10 @@ impl BurnchainSigner {
         }
     }
 
+    pub fn to_address(&self) -> String { 
```

Comment:
I don't think this is the address -- the address should be something more like:

```rust
let addr_type = match &self.hash_mode {
    SerializeP2PKH | SerializeP2WPKH => PublicKeyHash,
    _ => ScriptHash,
};
BitcoinAddress::from_bytes(BitcoinNetworkType::Testnet,
                                              addr_type, &btc_input.to_address_bits()).unwrap().to_string()
```

Reply:
+1 to @kantai

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/1562/commits/ba7811e182acbd7c661f8c984cd66370fe7dae6c